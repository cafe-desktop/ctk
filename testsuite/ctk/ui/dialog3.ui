<!-- invalid dialog markup attribute -->
<interface>
  <object class="CtkDialog">
    <child type="action">
      <object class="CtkButton" id="button_ok">
        <property name="can-default">True</property>
      </object>
    </child>
    <action-widgets>
      <action-widget response="warbl">button_ok</action-widget>
    </action-widgets>
  </object>
</interface>
