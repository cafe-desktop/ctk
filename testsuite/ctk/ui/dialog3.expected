ERROR: ctk-builder-error-quark 6
.:10:40 Could not parse enum: 'warbl'
